src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsaccreditedprogrammesapi/service/PersonService.kt

```diff
@@ -109,31 +109,39 @@ class PersonService(
   }
 
   enum class KeyDateType(val mapping: String, val code: String, val description: String, val order: Int = 1) {
-    ACTUAL_PAROLE_DATE("actualParoleDate", "APD", "Approved parole date"),
-    AUTOMATIC_RELEASE_DATE("automaticReleaseDate", "ARD", "Automatic release date"),
-    CONDITIONAL_RELEASE_DATE("conditionalReleaseDate", "CRD", "Conditional release date"),
+    ACTUAL_PAROLE_DATE("actualParoleDate", "APD", "Approved parole date", 1),
+    AUTOMATIC_RELEASE_DATE("automaticReleaseDate", "ARD", "Automatic release date", 2),
+    CONDITIONAL_RELEASE_DATE("conditionalReleaseDate", "CRD", "Conditional release date", 3),
```

Comment:
Minor: The order can we have increments of 10

If someone wants to change the order or have something in between actualParoleDate and automaticReleaseDate then that can be easily changed without having to change other records

Reply:
yeah can do